index.html

```diff
@@ -5,226 +5,148 @@
-                <img class="pic" src="./pics/works img/work1.svg" alt="card1" />
-            </div>
-            <div class="works-description">
-                <h1 class="comp-title">Tonic</h1>
-                <ul class="work-title">
-                    <li class="client">CANOPY</li>
-                    <li><img src="pics/works img/Counter.svg" alt="dot" /></li>
-                    <li class="dim">Back End Dev</li>
-                    <li><img src="pics/works img/Counter.svg" alt="dot" /></li>
-                    <li class="dim">2015</li>
-                </ul>
-                <div class="project-desc">
-                    <p>A daily selection of privately personalized reads; <br>no accounts or sign-ups required.
-                    </p>
-                </div>
-                <ul class="langs">
-                    <li class="tag">html</li>
-                    <li class="tag">css</li>
-                    <li class="tag">javaScript</li>
-                </ul>
-                <button class="btn">See Project</button>
-            </div>
-        </div>
-        <div class="card reverse">
-            <div class="portfolio-img pic2">
-                <img class="pic" src="./pics/works img/work4.svg" alt="card1" />
-            </div>
-            <div class="works-description">
-                <h1 class="comp-title">Multi-Post
-                    Stories</h1>
-                <ul class="work-title">
-                    <li class="client">FACEBOOK</li>
-                    <li><img src="pics/works img/Counter.svg" alt="dot" /></li>
-                    <li class="dim">Full Stacks Dev</li>
-                    <li><img src="pics/works img/Counter.svg" alt="dot" /></li>
-                    <li class="dim">2015</li>
-                </ul>
-                <div class="project-desc">
-                    <p>Experimental content creation feature that allows users to add to an existing story over the
-                        course of a day without spamming their friends.</p>
+    <main id="root">
+        <header>
+            <nav class="navbar">
+                <div id="logo">
+                    <a href="#index"><img src="./images/MyLogo.png" alt="logo" /></a>
                 </div>
-                <ul class="langs">
-                    <li class="tag">html</li>
-                    <li class="tag">Ruby on rails</li>
-                    <li class="tag">css</li>
-                    <li class="tag">javaScript</li>
+                <div class="openMenu"><i class="fa fa-bars"></i></div>
+                <ul class="nav-items">
+                    <li class="portfolio item"><a href="#workscontainer" class="link">Portfolio</a></li>
+                    <li class="about item"><a href="#about-container" class="link">About</a></li>
+                    <li class="contact item"><a href="#contact-container" class="link">Contact</a></li>
+                    <li class="closeMenu"><i class="fa fa-times"></i></li>
                 </ul>
```

Comment:
- [ ] You did a great job 😃 at adding the time icon of the mobile menu, but I see that the style of the times icon needs to stick to the Figma Design, as you see here the font size of the bars is greater than the Figma, so please 😊 try to make it like the Figma to be excellent for the eye💯


> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/224783211-ae479c53-2ddb-4eff-84c2-197c02e15be3.png)   |   ![image](https://user-images.githubusercontent.com/83360636/224783159-18110d00-ecdf-4b79-8a58-831a89385c33.png) |